examples/cn0554_example.py

```diff
@@ -0,0 +1,948 @@
+#!/usr/bin/python3
+
+import sys
+import os
+import time
+import csv
+
+from PyQt5.QtCore import Qt, QRect, QThread, pyqtSlot, pyqtSignal
+from PyQt5.QtWidgets import *
+#from PyQt5.QtChart import *
+from PyQt5 import QtGui
+from PyQt5.QtGui import *
+
+import pyqtgraph
+import pyvisa
+
+def readBuffer(buffer_loc):
+	buffer_pipe = None
+	try:
+		if os.path.exists(buffer_loc):
+			buffer_pipe = os.popen('cat ' + buffer_loc)
+			output = buffer_pipe.read()
+		else:
+			raise Exception('Path:' + buffer_loc + ' does not exist')
+	except Exception as e:
+		print(str(e))
+		if buffer_pipe is not None:
+			buffer_pipe.close()
+		return False
+	buffer_pipe.close()
+	return output
+
+def writeBuffer(wstring, buffer_loc):
+	buffer_pipe = None
+	try:
+		if os.path.exists(buffer_loc):
+				buffer_pipe = os.popen("echo " + str(wstring) + " > " + buffer_loc)
+		else:
+			raise Exception('Path:' + buffer_loc + ' does not exist')
+	except Exception as e:
+		print(str(e))
+		if buffer_pipe is not None:
+			buffer_pipe.close()
+		return False
+	buffer_pipe.close()
+	return True
+
+class AD7124():
```

Comment:
Why are there re-implementation of these classes here? (AD7124, LTC2688, CN0544)